app/Http/Controllers/CityController.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\Models\City;
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\Validator;
+
+class CityController extends Controller
+{
+    public function index()
+    {
+        return view('cities.index');
+    }
+
+    public function getAll()
+    {
+        $cities = City::all();
+        return response()->json(['cities' => $cities]);
+    }
+
+    public function store(Request $request): JsonResponse
```

Comment:
You could use FormRequest to define all the rules. Then you dont need to care about validaror->fails() for example. 

You can read more about FormRequests here https://laravel.com/docs/11.x/validation#form-request-validation